skbeam/core/accumulators/projector.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python
+
+from __future__ import division
+import numpy as np
+
+"""
+authors: Mikhail Dubrovin, TJ Lane, Christopher O'Grady
+"""
+
+
+class Projector(object):
+
+    def __init__(self, bin_values, nbins, weights=None, norm=True):
+        """
+        Parameters
+        ----------
+        bin_values : np.ndarray
+            For each pixel, this is the value of that determines which bin
```

Comment:
Is this an array of ints such that `0 <= bv < nbins` for all `bv in bin_values` or an array of arbitrary floats which will be made discrete in the function call?